Tests/Recurly/ShippingAddress_Test.php

```diff
@@ -24,6 +20,25 @@ public function testCreateShippingAddressOnExistingAccount() {
     $shad->state = "CA";
     $shad->zip = "94110";
     $shad->country = "US";
+    $shad->company = "Recurly Inc.";
+
+    return $shad;
+  }
+
+  public function testXml() {
+    $shad = $this->mockShippingAddress();
+
+    $this->assertEquals(
+      "<?xml version=\"1.0\"?>\n<shipping_address><address1>123 Dolores St.</address1><city>San Francisco</city><state>CA</state><zip>94110</zip><country>US</country><phone>555-555-5555</phone><email>verena@example.com</email><nickname>Work</nickname><first_name>Verena</first_name><last_name>Example</last_name><company>Recurly Inc.</company></shipping_address>\n",
+      $shad->xml()
+    );
+  }
```

Comment:
👍  now i get a failing spec if i comment out the other branch in `Recurly_ShippingAddress::populateXmlDoc()`